<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="ControlPanelWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="CenterDownSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QSplitter" name="LeftRightPartSplitter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <widget class="QSplitter" name="LeftPartSplitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>0</number>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="BatFlagsSignalsGrid" rowstretch="3,0" columnstretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="QWidget" name="FlagsWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="SignalsWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QWidget" name="BatInfoWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="InfChargeLayout">
               <item>
                <layout class="QVBoxLayout" name="GenInfoTxtLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="VoltageLabel">
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Напряжение:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="CurrentLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Ток:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="CurCapacityLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Текущая ёмкость батерии:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="MaxCapacityLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Максимальная ёмкость батареи:   </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="BatteryChargeImg">
                 <property name="styleSheet">
                  <string notr="true">QLabel {
	border: 0px;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="build/Desktop_Qt_6_7_2_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc">:/images/BatteryIcon.png</pixmap>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="VoltageTempInfoLayout">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="VoltageInfoLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="MaxVoltageLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Текущий max напряжения:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="MinVoltageLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Текущий min напряжения:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="VoltageDIfLabel">
                   <property name="font">
                    <font>
                     <family>Roboto</family>
                     <pointsize>13</pointsize>
                     <italic>false</italic>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Разница напряжений:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="TempInfoLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="MaxTempLabel">
                   <property name="font">
                    <font>
                     <family>Roboto</family>
                     <pointsize>13</pointsize>
                     <italic>false</italic>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Текущий max температуры:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="MinTempLabel">
                   <property name="font">
                    <font>
                     <family>Roboto</family>
                     <pointsize>13</pointsize>
                     <italic>false</italic>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Текущий min температуры:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="TempDifLabel">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
	color: rgb(25, 25, 25);
	border: 0px;
	background-color: transparent;
	font: 13pt &quot;Roboto&quot;;
}</string>
                   </property>
                   <property name="text">
                    <string>Температурная девиация:
</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QSplitter" name="RightPartSplitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="handleWidth">
         <number>0</number>
        </property>
        <property name="childrenCollapsible">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="VoltCellsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
         </property>
         <widget class="QProgressBar" name="ProgressBar">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>90</y>
            <width>70</width>
            <height>70</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>100</horstretch>
            <verstretch>70</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar {
	border: 0px;
	border-radius: 0px;
	background: transparent;
	alternate-background-color: rgb(85, 255, 0);
}
QProgressBar::chunk {
	border-radius: 0px;
	background:rgb(85, 255, 0);
}</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="TempCellsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="FooterWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
background-color: rgb(250, 250, 250); 
border: 1px solid rgb(150, 150, 160);
}</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="SettingsMenu">
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="ViewMenu">
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="SettingsMenu"/>
   <addaction name="ViewMenu"/>
  </widget>
 </widget>
 <resources>
  <include location="build/Desktop_Qt_6_7_2_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc"/>
 </resources>
 <connections/>
</ui>
